t/tdigest/tdigest_ubi_8.5.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash -e
+# -----------------------------------------------------------------------------
+#
+# Package       : tdigest
+# Version       : v0.0.0-20181121200506-bf2b5ad3c0a9,v0.0.1
+# Source repo   : https://github.com/influxdata/tdigest.git
+# Tested on     : ubi 8.5
+# Language      : go
+# Travis-Check  : true
+# Script License: Apache License, Version 2 or later
+# Maintainer    : Ambuj Kumar {Ambuj.Kumar3@ibm.com}
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+#Run the script:./tdigest_ubi_8.5.sh v0.0.1(version_to_test)
+PACKAGE_NAME=tdigest
+#PACKAGE_VERSION=${1:-v0.0.1}
+PACKAGE_VERSION=${1:-bf2b5ad3c0a9}
+GO_VERSION=1.16.1
+PACKAGE_URL=https://github.com/influxdata/tdigest.git
+
+dnf install git wget sudo make gcc gcc-c++ -y
+
+mkdir -p /home/tester/output
+cd /home/tester
+
+wget https://golang.org/dl/go$GO_VERSION.linux-ppc64le.tar.gz
+rm -rf /home/tester/go && tar -C /home/tester -xzf go$GO_VERSION.linux-ppc64le.tar.gz
+rm -f go$GO_VERSION.linux-ppc64le.tar.gz
+export GOPATH=/home/tester/go
+export PATH=$PATH:$GOPATH/bin
+export  GO111MODULE=on
+mkdir -p $GOPATH/src/github.com/influxdata
+cd $GOPATH/src/github.com/influxdata
+rm -rf $PACKAGE_NAME
+
+
+OS_NAME=$(cat /etc/os-release | grep ^PRETTY_NAME | cut -d= -f2)
+
+
+if ! git clone $PACKAGE_URL $PACKAGE_NAME; then
+        echo "------------------$PACKAGE_NAME:clone_fails---------------------------------------"
```

Comment:
Please use uniformed indentation in the file.

Reply:
Hi Prashant done the review changes